tests/integration/gcp_secret_manager_provider_test.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+
+from agent_guard_core.credentials.gcp_secrets_manager_provider import GCPSecretsProvider
+from agent_guard_core.credentials.secrets_provider import SecretProviderException
+
+
+@pytest.fixture()
+def provider(scope="module"):
+    return GCPSecretsProvider(project_id="mela-50d4f",secret_id="oryantest")
```

Comment:
Can you please update the initialization of the provider to be more generic?
```suggestion
    return GCPSecretsProvider(project_id="<project ID>")
```